src/git/mod.rs

```diff
@@ -23,24 +23,26 @@ pub use utils::{tmp_dir, try_get_branch_from_path};
 
 // Assumptions:
 // * `--git <url>` should only be parse in the same way as `git clone <url>` would
-// * submodules should be clone by default
+// * submodules can be cloned by setting the submodules field to true.
```

Comment:
Are you certain the text is actually right?

I have understood that `--skip-submodules=false` will lead to checked out submodules (the default case) 
and `--skip-submodules=true` will not check them out.

Or do I miss anything?